tools/mo/openvino/tools/mo/utils/ir_reader/extenders/parameter_extender.py

```diff
@@ -24,8 +24,10 @@ def extend(op: Node):
             for i, dim in enumerate(op.shape):
                 if dim == -1 or (isinstance(dim, str) and ".." in dim):
                     shape[i] = -1
-                    if ".." in dim:
-                        has_shapes_with_boundaries = True
+                    # Check only if dim is not int
+                    if not isinstance(dim, int):
```

Comment:
Not a big deal but you can put both those conditions in a single `if` statement.